ingest/indexer/service/client/pubsub_client.go

```diff
@@ -116,6 +119,21 @@ func (p *PubSubClient) PublishTokenSupplyOffset(ctx context.Context, tokenSupply
 	return p.publish(ctx, tokenSupplyOffset, p.tokenSupplyOffsetTopicId)
 }
 
+// PublishPair implements PubSubClient.PublishPair
+func (p *PubSubClient) PublishPair(ctx context.Context, pair indexerdomain.Pair) error {
+	// Check if project id and topic id are set
+	if p.projectId == "" || p.pairTopicId == "" {
+		return errors.New("project id and pool topic id must be set")
+	}
+
+	if pair.IdxDenom0 == pair.IdxDenom1 {
```

Comment:
It is already checked in `PublishPoolPairs` in `pair_publisher.go` and so I guess it may be redundant?

Reply:
removed